### PR TITLE
dependency and linting updates

### PR DESCRIPTION
**Pull request by adamshire123**
#### What does this PR do?

updates dependencies
fixes linting problems

#### Includes new or updated dependencies?

YES

#### Developer

- [ ] All new ENV is documented in README
- [ ] Stakeholder approval has been confirmed (or is not needed)

#### Code Reviewer

- [ ] The commit message is clear and follows our guidelines (not just this pull request message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The documentation has been updated or is unnecessary
- [ ] The changes have been verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -179,68 +194,68 @@
...
             "hashes": [
-                "sha256:8d36afa7616d8ab714608411b4a3b13e58f463aee519024578e062e141dce20f",
-                "sha256:8f135f6502756bde6b2a9b28989df5fbe87c9970cecaa69041edcce7f0589b14"
+                "sha256:c97dfde1f7bd43a71c8d2a58e369e9b2bf692d1334ea9f9cae55add7d0dd0f84",
+                "sha256:fdb6d215c776278489906c2f8916e6e7d4f5a9b602ccbcfdf7f016fc8da0596e"
             ],
-            "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'",
-            "version": "==1.26.16"
+            "markers": "python_version >= '3.7'",
+            "version": "==2.0.7"
```
Comment (ghukill) [inline Pipfile.lock:201]: Given this major version change for `urllib3` from `1.26.16` to `2.0.7`, assuming all the tests ran okay?

**Review by ghukill — APPROVED** [review]
Looks good to me.

Left a comment about the major version bump for `urllib3` going from version `1.26.16` to `2.0.7`, but assuming tests passed, imagine that's okay.

File: Pipfile.lock (hunk 1 above)
Comment (ghukill) [reply Pipfile.lock:201]: As noted in slack, not seeing anything in the `2.x` changelog that would suggest this version bump would be problematic: https://github.com/urllib3/urllib3/blob/main/CHANGES.rst#L88-L114.

Comment (adamshire123) [reply Pipfile.lock:201]: all of the extra integration tests passed

**Pull request merged**